### PR TITLE
passwordless ssh for vios

### PR DESCRIPTION
**Pull request by smruti77**
passwordless ssh vios for network_virtualization, veth_dlpar and
dlpar_vscsi test. and add ip configuration in veth_dlpar test.
Signed-off-by: bismurti bidhibrata pattajoshi <bbidhibr@in.ibm.com>

### CODE REVIEW

**Comment by smruti77** [comment]
[dlpar_vscsi.log](https://github.com/avocado-framework-tests/avocado-misc-tests/files/4687598/dlpar_vscsi.log)
[network_virtualization.log](https://github.com/avocado-framework-tests/avocado-misc-tests/files/4687599/network_virtualization.log)
[veth_dlpar.log](https://github.com/avocado-framework-tests/avocado-misc-tests/files/4687602/veth_dlpar.log)

**Comment by narasimhan-v** [comment]
@smruti77 Thanks for this. I will wait for @abdhaleegit comments also, since this is involving network_virtualisation test

File: io/net/virt-net/veth_dlpar.py [hunk 1]
```diff
@@ -56,99 +37,65 @@ def setUp(self):
...
+        self.session = Session(self.vios_ip, user=self.vios_user,
+                               password=self.vios_pwd)
+        local = LocalHost()
+        self.networkinterface = NetworkInterface(self.interface, local)
+        try:
+            self.networkinterface.add_ipaddr(self.ipaddr, self.netmask)
+            self.networkinterface.save(self.ipaddr, self.netmask)
+        except Exception:
+            self.networkinterface.save(self.ipaddr, self.netmask)
+        self.networkinterface.bring_up()
```
Comment (narasimhan-v) [inline io/net/virt-net/veth_dlpar.py:56]: Please split this patch into 2 commits, one for IPAddr config, and one for vios command handling

File: io/net/virt-net/veth_dlpar.py.data/veth_dlpar.yaml [hunk 2]
```diff
@@ -1,4 +1,6 @@
 interface: ""
+host_ip: ""
+netmask: ""
```
Comment (narasimhan-v) [inline io/net/virt-net/veth_dlpar.py.data/veth_dlpar.yaml:3]: @smruti77 this should also go into the other commit

File: io/net/virt-net/veth_dlpar.py [hunk 3]
```diff
@@ -56,25 +37,28 @@ def setUp(self):
         Gather necessary test inputs.
         '''
         self.interface = self.params.get('interface', default=None)
+        self.ipaddr = self.params.get("host_ip", default="")
+        self.netmask = self.params.get("netmask", default="")
```
Comment (narasimhan-v) [inline io/net/virt-net/veth_dlpar.py:41]: this should also go into the other commit, right ?

**Pull request closed without merging**